addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -445,6 +445,7 @@ export class OdooEditor extends EventTarget {
             this.observer = new MutationObserver(records => {
                 records = this.filterMutationRecords(records);
                 if (!records.length) return;
+                this.dispatchEvent(new Event('contentChanged'));
```

Comment:
I'm not sure I understand the link between this change and the checklist feature ?

Reply:
This is explained in the commit message, maybe I need to clarify it then. The feature depends on a callback on the `contentChanged` event, which was not triggered anymore since the new editor was introduced. I fix that here.

Reply:
My bad, I did not read it.